### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by AndreaM2429**
This time I design the desktop version, for this task I implemented 

1. Media query (width >= 768px).
2. Change the position of some elements accordign to Figma template.
3. Update the information in the README about the features.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -561,16 +572,23 @@ input[type="submit"] {
     position: unset;
   }
 
-  .menu {
+  .menu,
+  .desc h2,
+  .mobim,
+  #barrita {
     display: none;
```
Comment (roshan-bajgain) [inline style.css:581]: - [x] According to the project requirement you should stick to the design as much as possible. Your navbar is different from Figma's design so please use the same color and font and also make font size similar to the design.
- [x] Also increase the font-weight for the logo to match the design
### Your design
his

### Figma design
fimga

File: index.html [hunk 2]
```diff
@@ -53,131 +53,142 @@ <h2>I'm Andrea.<br>Glad to see you!</h2>
...
+            <li class="lisec">
+              Back End Dev
+            </li>
+            <li>
+              <img src="Img/Counter.png" alt="counter">
+            </li>
+            <li class="lisec">
+              2015
+            </li>
+          </ul>
```
Comment (roshan-bajgain) [inline index.html]: - [x] I would suggest you please stick to the design as much as possible like color, font, etc. I notice that `Back end dev and 2015` has different color please stick to the design. Also, increase the font weight of the Tonic heading.
- [x] The color use is `color: #6B778C;` for the list.
- [x] Also I would suggest you please make changes on all the cards.
### Your design
his

### Figma design
figma

File: style.css [hunk 3]
```diff
@@ -612,15 +630,143 @@ input[type="submit"] {
...
+  .direc {
+    flex-direction: row-reverse;
+  }
+
+  .aboutinfo {
+    display: flex;
+    flex-direction: row;
+    padding-left: 0;
+    padding-right: 0;
+  }
```
Comment (roshan-bajgain) [inline style.css]: - [x] You should stick to the design as much as possible like color, font, spacing, etc. In the About section, the left one takes 40% and the right one the remaining 60% of the page's width. Please do accordingly. Also please update the color of language, framework, and skills, the color use is `color: #344563;`.
- [x] Also I would suggest please increase the box size for javascript, HTML and CSS and the spacing for it. Pleas see image below for reference
### Your design
his

### Figma design
figma

File: index.html [hunk 4]
```diff
@@ -255,10 +266,10 @@ <h2>Contact me</h2>
         <input type="email" name="email" id="email" placeholder="email" required>
         <textarea name="message" id="message" cols="30" rows="10" placeholder="Write your message here..."
           maxlength="500" required></textarea>
-        <input class="efecbot" type="submit" placeholder="Get in in touch">
+        <input class="efecbot" type="submit" value="Get in touch">
       </form>
     </div>
```
Comment (roshan-bajgain) [inline index.html:271]: - [x] I would suggest you please remove the border of Get in touch to match the Figma design. Also provide the border-radius to match the design
### Your design
his

### Figma design
figma

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @AndreaM2429, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
- [ ] Please implement all the changes given by me so It works for mobile version as well.
- [ ] [Optional] I cannot leave comment there so I am leaving it here please there is a space in My self so please make it About Myself and also increse the font-weight of About Myself heading.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (AndreaM2429) [reply style.css:581]: Hello @roshan-bajgain my nav looks like this now
image

File: style.css (hunk 3 above)
Comment (AndreaM2429) [reply style.css]: Hello @roshan-bajgain this looks like this now
image

Comment (AndreaM2429) [reply style.css]: @roshan-bajgain Also about the padding I removed those line of code and continue working with the last padding
image
Is it ok?

**Review by Shisui6 — APPROVED** [review]
Hi @AndreaM2429, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights
- Good use of media queries ✅
- Nice design ✅


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**